src/cli.rs

```diff
@@ -29,20 +19,10 @@ pub struct Cli {
 
 impl Cli {
     pub fn run(&self) -> Result<()> {
-        let rbt: Rbt = match &self.load_from_json {
-            Some(path) => Self::load_from_json(path).context("could not load from JSON")?,
-            None => Self::load_from_roc(),
-        };
-
-        if self.dump_to_json {
-            println!(
-                "{}",
-                serde_json::to_string(&rbt).context("could not dump to JSON")?
-            )
-        }
+        let rbt = Self::load();
 
         let mut coordinator = Coordinator::default();
-        coordinator.add_target(&rbt.default);
+        coordinator.add_target(rbt.f0.default);
```

Comment:
Can this get better naming? `rbt.f0.default` does not feel to have much meaning.